Assets/Scripts/PlayerController.cs

```diff
@@ -1,15 +1,23 @@
 ﻿using System.Linq;
 using UnityEngine;
-using Cinemachine;
 
 public class PlayerController : MonoBehaviour
 {
     public bool EnablePlayerInput { get; set; } = true;
     public bool controllerConnected = false;
 
-    void Start()
+    [HideInInspector] public Camera mainCamera;
```

Comment:
We can just do `CameraManager.Instance.brain.gameObject` to get access to the MainCamera, without having to have a reference to it or worry about finding it.